lasagne/layers/conv.py

```diff
@@ -930,9 +932,211 @@ def convolve(self, input, **kwargs):
         conved = op(self.W, input, output_size)
         return conved
 
+
```

Comment:
The idea was to have the aliases closer to the class definition (with a single empty line) than to other class definitions (two empty lines). pep8 was happy with it.